src/main/java/org/folio/linked/data/validation/dto/LccnValidator.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.linked.data.validation.dto;
+
+import jakarta.validation.ConstraintValidator;
+import java.lang.annotation.Annotation;
+
+public abstract class LccnValidator<A extends Annotation, T> implements ConstraintValidator<A, T> {
```

Comment:
Any need to keep this interface? May be remove it if it is not needed any more?

Reply:
done.